.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - 1.x
+  - 1.7.x
+  - master
+
+install: make get-deps
+script: make build
```

Comment:
Right now, because there is no test suite, I'm just making sure the build passes for every platform so as to prevent errors like the one I'd made.